src/menus/FileMenus.cpp

```diff
@@ -553,12 +549,16 @@ auto FileMenu()
          )
       ),
 
+      Section( "Close", Command( wxT("Close"), XXO("&Close Project"), OnClose,
+            AudioIONotBusyFlag(), wxT("Ctrl+W") )
+      ),
+
       Section( "Exit",
          // On the Mac, the Exit item doesn't actually go here...wxMac will
          // pull it out
          // and put it in the Audacity menu for us based on its ID.
          /* i18n-hint: (verb) It's item on a menu. */
```

Comment:
That said, watch the comments. If the entire section is relocated to the App menu on macOS, you'll need to do some platform specific code via

```cpp

#ifdef __WXMAC__
  macos code
#else 
  windows/linux code
#endif
```

Reply:
I am not really familiar with mac but what I am understanding is that if I put both 'Quit Audacity' and 'Close Project' in the same section then to make sure that mac doesn't put 'Close Project' in the app menu as well, they would need to be in different sections. Won't that result in the section dividing line still being present in mac then?